src/plugins/python/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ add_plugin(python
 if (ADDTESTING_PHASE)
 	# bindings are required for tests
 	list (FIND BINDINGS "swig_python" FINDEX)
-	if (NOT BUILD_SHARED AND BUILD_TESTING AND FINDEX GREATER -1)
```

Comment:
Looks like this update causes a segfault in [`elektra-gcc-configure-debian-log`](https://build.libelektra.org/jenkins/job/elektra-gcc-configure-debian-log/971/console) and errors in the main [Mac builds](https://travis-ci.org/ElektraInitiative/libelektra/jobs/314400162) when Elektra is installed.